descriptors/ModuleDescriptor-template.json

```diff
@@ -200,6 +200,46 @@
       "version": "5.3"
     }
   ],
+  "metadata": {
+    "user": {
+      "type": "system",
+      "permissions": [
```

Comment:
Is there any way to share these with the file in src/main/resources? It seems like it will be a headache to keep both places up to date.

Reply:
No, it's not possible to do with current build steps and tools

I can suggest discussing this improvement with the team responsible for https://github.com/folio-org/folio-spring-support/tree/master/folio-spring-system-user